### ISSUE TITLE
Force no space between properties

### ISSUE THREAD

**Issue by gajus** [eslint/eslint#3841]
```js
/**
 * Change the status of component in state.
* 
 * @typedef changeStatus
 * 
 * @property {status} String
 * @return {Boolean}
 */
```

Have a rule to detect a space between properties.

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by nzakas** [comment]
This isn't enough information to evaluate. Please read the documentation referenced in the previous comment and update with more information.

**Comment by gajus** [comment]
Will re-open in a couple of days with a lot more information.
